apps/picsa-tools/resources-tool/src/app/services/resources-tool.service.ts

```diff
@@ -71,6 +73,8 @@ export class ResourcesToolService extends PicsaAsyncService {
   }
 
   public async openFileResource(uri: string, mimetype: string) {
+    // track the resource open event
+    this.analyticsService.trackResourceOpen();
```

Comment:
nit(blocking): Nice we can track the resource open event but would be even better if we could track the id of the resource being opened. So in order to achieve this we either need to pass an extra argument to the `openFileResource` method (e.g. `id`), or call the tracking function within the `resource-item-file` component itself which does have the id.

I think technically the better solution would probably be passing the id to the service, as that keeps the component more focused on the UI and service on side-effects